.npmrc

```diff
@@ -1 +1 @@
-package-lock=false
```

Comment:
I'm not sure we can use package-lock with our version of lerna. There were some issues a while back that prevented us from using it. Need to look into this some more.